### ISSUE TITLE
Microsoft edge web worker returns 'no available storage method found'

### ISSUE THREAD

**Issue by nirbil** [localForage/localForage#863]
I'm trying to use localForage within a dedicated web worker using indexedDB storage.
The web worker is created from a Blob URI.
On Microsoft edge i'm getting a 'no available storage method found' error.
If I run the same code within the regular thread, everything seems to be working fine.

I've found the following bug report but I'm not entirely sure that is the root cause of the problem:
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/5942817/

I'm not sure why does the driver initialization fails.
I've tried testing myself whether the worker has the relevant APIs, and by that decide on-the-fly which thread to use, but my test seems to be returning true even on Edge within a Web Worker...
Here's my test code, its a snippet similar to the localForage's IDB driver test:

	var idbAccess = 
		typeof indexedDB !== 'undefined' ||
		typeof webkitIndexedDB !== 'undefined' ||
		typeof mozIndexedDB !== 'undefined' ||
		typeof OIndexedDB !== 'undefined' ||
		typeof msIndexedDB !== 'undefined';
	var isSafari =
		typeof openDatabase !== 'undefined' &&
		/(Safari|iPhone|iPad|iPod)/.test(navigator.userAgent) &&
		!/Chrome/.test(navigator.userAgent) &&
		!/BlackBerry/.test(navigator.platform);
	var hasFetch =
		typeof fetch === 'function' &&
		fetch.toString().indexOf('[native code') !== -1;
	var res = 
		idbAccess &&
		(!isSafari || hasFetch) &&
		typeof IDBKeyRange !== 'undefined';

So basically I'm trying to understand whether this is a bug in localForage and IDB storage should work in this case, or this is a bug in Edge and if that is the case how would I go about and test it so I could implement a sort of progressive enhancement.

**Comment by demsey2** [comment]
hi @nirbil, this is quite an old issue. I have a similar problem. Did you fix that?
